src/SFA.DAS.FindApprenticeshipTraining.Api.UnitTests/Models/WhenBuildingGetProviderFeedbackAttributes.cs

```diff
@@ -69,9 +68,9 @@ public void Then_No_Strengths_Returns_Empty_List(GetProvidersListItem source, st
             
             var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1,new List<DeliveryModeType>(),new List<FeedbackRatingType>(), true);
 
-            response.Feedback.FeedbackAttributes.Strengths.Should().BeEmpty();
-            response.Feedback.FeedbackAttributes.Weaknesses.Should().Contain(source.FeedbackAttributes.Select(c => c.AttributeName).ToList());
-            response.Feedback.FeedbackAttributes.Weaknesses.Should().NotBeEmpty();
+            response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Should().BeEmpty();
+            response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Should().Contain(source.FeedbackAttributes.Select(c => c.AttributeName).ToList());
+            response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Should().NotBeEmpty();
```

Comment:
These tests need to be looked at in more detail. The previous iterations of the unit tests relied on the fact there was two collections, a strength and a weakness. Now there is only one and the assertions completely contradict each other so assume this is a renaming gone wrong. But the build is failing due to these unit tests



` response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Should().BeEmpty();
 response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Should().NotBeEmpty();`

Reply:
Updated tests & renamed according to changes made